src/esp/scene/SceneGraph.h

```diff
@@ -27,12 +27,12 @@ class SceneGraph {
   SceneNode& getRootNode() { return rootNode_; }
   const SceneNode& getRootNode() const { return rootNode_; }
 
-  // TODO: remove this
-  gfx::DrawableGroup& getDrawables() {
-    return drawableGroups_.at(std::string{});
+  gfx::DrawableGroup& getDrawables(const std::string& groupName = "") {
```

Comment:
```suggestion
  gfx::DrawableGroup& getDrawables(const std::string& groupName = {}) {
```

`std::string` initialization from an empty C string literal involves a lot of nasty code including a `strlen()`, calling the default constructor not.

Same below.

Reply:
Yeah, good to know.
I was thinking the empty string was OK, since `strlen()` returned 0.